README.md

```diff
@@ -194,8 +194,9 @@ const { flags, values, positionals } = parseArgs(argv, options);
   - If `--` signals the end, is `--` included as a positional?  is `program -- foo` the same as `program foo`?  Are both `{positionals:['foo']}`, or is the first one `{positionals:['--', 'foo']}`?
 - Does the API specify whether a `--` was present/relevant?
   - no
-- Is `-foo` the same as `--foo`?
-  - no, `-foo` is a short option or options (WIP: https://github.com/pkgjs/parseargs/issues/2)
+- Is `-bar` the same as `--bar`?
+  - no, `-bar` is a short option or options, with expansion logic that follows the
```

Comment:
Are you thinking we will or won't do combined short option and value like `-p80`, or currently undecided?

Reply:
I was undecided for right now. Would it be sufficient to special case integers for this type of option?